unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowController.cs

```diff
@@ -268,37 +311,92 @@ private void HandleViewFocused(bool focused)
         {
             deactivatePreviewCancellationToken.Cancel();
             deactivatePreviewCancellationToken = new CancellationTokenSource();
-            DeactivatePreviewMode();
+            deactivatePreviewHUDControllerCancellationToken.Cancel();
+            deactivatePreviewHUDControllerCancellationToken = new CancellationTokenSource();
+            //Means something is visible. Restarting the counters is enough
+            if (_currentState.Equals(ChatWindowVisualState.INPUT_MODE) || _currentState.Equals(ChatWindowVisualState.PREVIEW_MODE))
+                return;
+
+            //We have to activate preview only for the ChatHudController and dont restart the timer
+            ActivatePreviewOnlyHUDController();
```

Comment:
i think its easier to follow if you do:
```
if (currentState == ChatWindowVisualState.NONE_VISIBLE)
    ActivatePreviewOnMessages()
````

on the other hand, why you have to activate it only on the messages? what if you call `ActivatePreview`?

Reply:
I made a different method since:
- `ActivatePreview` involves activating the preview on the input window. In this case, it wont do anything, since its going to start a function that goes from alpha 0 to alpha 0. By not calling `ActivatePreview`, we save doing that function
- `ActivatePreviewOnlyHUDController` doesn't start the timer to fade out preview mode.

A way to use only one method would be to add a bool parameter to `ActivatePreview` which describes if the timer should be started or no